extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +110,94 @@
     opacity: .4;
   }
 }
+
+/* Disclaimer CSS rules */
+.i-amphtml-story-interactive-disclaimer {
+  position: absolute !important;
+  color: black !important;
+  bottom: calc(100% - 1.75em) !important;
+  right: 0 !important;
+  z-index: 1 !important;
+}
+
+.i-amphtml-story-interactive-container:not(.i-amphtml-story-interactive-active) .i-amphtml-story-interactive-disclaimer {
+  display: none !important;  /* Disable the disclaimer if not on active page to prevent tabbing */
+}
+
+.i-amphtml-story-interactive-disclaimer-alert {
+  width: 1.75em !important;
+  height: 1.75em !important;
+  position: absolute !important;
+  right: 0 !important;
+  bottom: 0 !important;
+  transform-origin: bottom right !important;
+  pointer-events: all !important;
```

Comment:
I wonder if we can remove this here and in `i-amphtml-story-interactive-disclaimer-bubble` and put it once on `i-amphtml-story-interactive-disclaimer`.